src/marqo/tensor_search/tensor_search.py

```diff
@@ -1508,21 +1512,32 @@ def vectorise_jobs(jobs: List[VectorisedJobs]) -> Dict[JHash, List[float]]:
     """ Run s2 inference on a set of vector jobs."""
     result: Dict[JHash, List[float]] = {}
     for v in jobs:
-        result[v.groupby_key()] = functools.reduce(
-            lambda x, y: x + y,
-            [s2_inference.vectorise(
-                model_name=v.model_name, model_properties=v.model_properties,
-                content=batch, device=v.device,
-                normalize_embeddings=v.normalize_embeddings,
-                image_download_headers=v.image_download_headers
-            ) for batch in v.content] # Todo: check if should be batching.
-        )
+        # TODO: Handle exception for single job, and allow others to run.
```

Comment:
What's the plan for this?

Reply:
Not in this PR. I don't think it's worth doing until your PR is added.